deploy/helm/stac-scaler/values.yaml

```diff
@@ -0,0 +1,155 @@
+jobCleanupTimeSeconds: 600
+activeDeadlineSeconds: 600
+
+deploymentNamespace: pgstac
+envCode: stac
+
+processors:
+  staccollection:
+    template: processor-stac-collection
+    labels: 
+      app: app-stac-collection
+    image:
+      repository: acr.azurecr.io
+      name: stac-cli
+      pullPolicy: Always
+      tag: "latest"
+    env:
+      SERVICE_BUS_CONNECTION_STRING: ""
+      TOPIC_NAME: "staccollection"
+      SUBSCRIPTION_NAME: "staccollectiontopic"
+      DATA_STORAGE_ACCOUNT_CONNECTION_STRING: ""
+      DATA_STORAGE_ACCOUNT_NAME: ""
+      DATA_STORAGE_ACCOUNT_KEY: ""
+      STACCOLLECTION_STORAGE_CONTAINER_NAME: ""
+      AZURE_LOG_CONNECTION_STRING: ""
+      PGHOST: ""
+      PGPORT: "5432"
+      PGUSER: ""
+      PGDATABASE: ""
+      PGPASSWORD: ""
+      WORKING_DIR: "/app"
+      PYTHONUNBUFFERED: 0
+    podAnnotations: {}
+    resources:
+      requests:
+        memory: "718Mi"
+      limits:
+        cpu: "100m"
+        memory: "1024Mi"
+  staceventconsumer:
+    template: processor-stac-event-consumer
+    labels: 
+      app: app-stac-event-consumer
+    image:
+      repository: acr.azurecr.io
+      name: stac-cli
+      pullPolicy: Always
+      tag: "latest"
+    env:
+      SERVICE_BUS_CONNECTION_STRING: ""
+      TOPIC_NAME: "pgstactopic"
+      SUBSCRIPTION_NAME: "pgstacsubscription"
+      DATA_STORAGE_ACCOUNT_CONNECTION_STRING: ""
+      GENERATED_STAC_STORAGE_CONTAINER_NAME: ""
+      AZURE_LOG_CONNECTION_STRING: ""
+      DATA_STORAGE_PGSTAC_CONTAINER_NAME: ""
+      PGHOST: ""
+      PGPORT: "5432"
+      PGUSER: ""
+      PGDATABASE: ""
+      PGPASSWORD: ""
+      MESSAGE_COUNT: 50
+      WORKING_DIR: "/app"
+      PYTHONUNBUFFERED: 0
+    podAnnotations: {}
+    resources:
+      requests:
+        memory: "1024Mi"
+      limits:
+        cpu: "100m"
+        memory: "1536Mi"
+  generatestacjson:
+    template: processor-generate-stac-json
+    labels:
+      app: app-generate-stac-json
+    image:
+      repository: acr.azurecr.io
+      name: stac-cli
+      pullPolicy: Always
+      tag: "latest"
+    env:
+      DATA_STORAGE_ACCOUNT_CONNECTION_STRING: ""
+      DATA_STORAGE_ACCOUNT_NAME: ""
+      DATA_STORAGE_ACCOUNT_KEY: ""
+      STACIFY_STORAGE_CONTAINER_NAME: ""
+      SERVICE_BUS_CONNECTION_STRING: ""
+      TOPIC_NAME: "stacify"
+      SUBSCRIPTION_NAME: "stacifytopic"
+      GENERATED_STAC_STORAGE_CONTAINER_NAME: ""
+      AZURE_LOG_CONNECTION_STRING: ""
+      DATA_STORAGE_PGSTAC_CONTAINER_NAME: ""
+      STAC_METADATA_TYPE_NAME: ""
+      JPG_EXTENSION: "200.jpg"
+      XML_EXTENSION: "aux.xml"
+      COLLECTION_ID: "naip"
+      MESSAGE_COUNT: 50
+      WORKING_DIR: "/app"
+      PYTHONUNBUFFERED: 0
+    podAnnotations: {}
+    resources:
+      requests:
+        memory: "1024Mi"
+      limits:
+        cpu: "100m"
+        memory: "1536Mi"
+
+stacfastapi:
+  replicas: 1
+  privateIp: 10.6.3.15
+  labels:
+    app: app-stacfastapi
+  image:
+    repository: acr.azurecr.io
```

Comment:
If a default is supplied, it should be usable. Otherwise, you can put it here, but commented out. What we want to avoid is someone deploying without the required vars set and then it doesn't work and is difficult to figure out why. In the case of required variables, there is a jinja filter called `require` (or maybe it's `required`--have to look) that will fail with a message if there's no value. You would apply that filter wherever this value is used. If it's used a bunch, you can make a template for it and call that.

Reply:
Addressed by not supplying a default value. Will explore jinja option in future PRs.